perfmetrics/scripts/compare_fuse_types_using_fio.py

```diff
@@ -0,0 +1,227 @@
+"""Python script to do FIO test for GCSFuse Vs GCSFuse/Goofys.
+
+To run the script:
+python3 compare_fuse_types_using_fio.py -- --fuse_type_1=<value1> --fuse_type_1_version=<version1> --flags_1=<flags1> 
+--fuse_type_2=<value2> --fuse_type_2_version=<version2> --flags_2=<flags2> --jobfile_path=<jobfile_path> --gcs_bucket=<gcs_bucket>
+
+1) --fuse_type_1=<value1> & --fuse_type_2=<value2>
+-> Value1 and value2 can be fuse based system name.
```

Comment:
i believe fuse file system name is used only for creating directory and logging. Then mention that also in comment like the name is used only in logging. so it can be anything.
But for gcsfuse it has be only gcsfuse.